src/sync/models/topic.js

```diff
@@ -14,27 +14,24 @@ class Topic {
   }
 
   decode() {
+    this.version = this.rawLog['_version'].toNumber();
+    this.topicAddress = this.rawLog['_topicAddress'];
+
     let nameHex = _.reduce(this.rawLog['_name'], (hexStr, value) => {
       return hexStr += value;
     }, '');
     this.name = Utils.toUtf8(nameHex)
+
     let intermedia = _.map(this.rawLog['_resultNames'], (item) => Utils.toUtf8(item));
     this.resultNames = _.filter(intermedia, item => !!item);
-
-    this.topicAddress = this.rawLog['_topicAddress'];
-    this.creator = this.rawLog['_creator'];
-    this.oracle = this.rawLog['_oracle'];
-    this.bettingEndBlock = this.rawLog['_bettingEndBlock'].toNumber();
-    this.resultSettingEndBlock = this.rawLog['_resultSettingEndBlock'].toNumber();
   }
 
   translate() {
     return {
       _id: this.topicAddress,
+      version: this.version,
       address: this.topicAddress,
       txid: this.txid,
```

Comment:
no more `creator`address?

Reply:
yes removed for now since contract does not use this and no need to return.